apps/extension/src/utils/taskFileReader.ts

```
@@ -0,0 +1,214 @@
+	subtasks?: TaskWithDetails[];
+}
+
+/**
+ * Reads tasks.json file directly and extracts implementation details and test strategy
+ * @param taskId - The ID of the task to read (e.g., "1" or "1.2" for subtasks)
+ * @param tagName - The tag/context name (defaults to "master")
+ * @returns TaskFileData with details and testStrategy fields
+ */
+export async function readTaskFileData(
+	taskId: string,
+	tagName: string = 'master'
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)